respokeSDK/src/main/java/com/digium/respokesdk/RestAPI/APITransaction.java

```diff
@@ -56,7 +58,18 @@
     protected String params;
     protected int serverResponseCode;
     private AsyncTransaction asyncTrans;
-    
+
+    public static String getSDKHeader() {
+        String sdkTitle = "Respoke-Android";
+        String sdkVersion = BuildConfig.VERSION_NAME;
```

Comment:
I'm unsure if BuildConfig.VERSION_NAME would match the sdk version when the sdk is being compiled into some other Android application.